src/helpers/api.js

```diff
@@ -93,7 +93,8 @@ const API = {
 
         const requestOptions = {
           method: "get",
-          url
+          url,
+          params: data.params || {}
```

Comment:
What is this used for? I don't see where `data` is passed in to `API.get(url, data)` anywhere.

Reply:
He added it to specify currency for the price route, but finally he chose to use query parameters instead of a data body. We should just discard these changes now.

Reply:
Exactly